src/modules/components/transaction/transactionDataListItem/transactionDataListItemStructure/transactionDataListItem.test.tsx

```diff
@@ -0,0 +1,64 @@
+import { render, screen } from '@testing-library/react';
+import { DataList } from '../../../../../core';
+import { TransactionDataListItemStructure } from './transactionDataListItemStructure';
+import {
+    TransactionType,
+    TxStatusCode,
+    type ITransactionDataListItemProps,
+} from './transactionDataListItemStructure.api';
+
+describe('<TransactionDataListItemStructure /> component', () => {
+    const createTestComponent = (props?: Partial<ITransactionDataListItemProps>) => {
+        return (
+            <DataList.Root entityLabel="Daos">
+                <DataList.Container>
+                    <TransactionDataListItemStructure {...props} />
+                </DataList.Container>
+            </DataList.Root>
+        );
+    };
+
+    it('renders the transaction type heading', () => {
+        const txType = TransactionType.ACTION;
+        render(createTestComponent({ txType }));
+        const transactionTypeHeading = screen.getByText('Smart contract action');
+        expect(transactionTypeHeading).toBeInTheDocument();
+    });
+
+    // fails on the local run, but passes on the CI for relative unix timestamps (CI server location == UTC 0)
+    it('renders the formatted date', () => {
+        const unixTimestamp = 1628841600;
+        render(createTestComponent({ unixTimestamp }));
+        const formattedDate = screen.getByText('January 19, 1970 at 8:27 PM');
```

Comment:
Shouldn't using the formatter in the test actually fix the local and CI time issue?
```suggestion
        const formattedDate = screen.getByText(formatDate(unixTimeStamp));
```
With this, you are just testting that the date is being formatted. You needn't worry about the actual format and value; those will be tested with the date formatter util's own unit tests.

That said, take a look at my comment about the formatter itself before you make any changes.